src/qt/locale/bitcoin_de.ts

```diff
@@ -3170,15 +3170,15 @@ Beispiel: alertnotify=echo %%s | mail -s "Bitcoin Alert" admin@foo.com
     </message>
     <message>
         <source>Copyright (C) 2009-%i The Bitcoin Core Developers</source>
-        <translation>Urheberrecht (C) 2009-%i Die "Bitcoin Core"-Entwickler</translation>
+        <translation>Urheberrecht (C) 2009-%i Die "Dogecoin Core"-Entwickler</translation>
```

Comment:
Current one says  

Copyright © 2009-2014 The Bitcoin Core developers
Copyright © 2013-2014 The Dogecoin developers